mscore/preferences.cpp

```diff
@@ -648,7 +648,7 @@ PreferenceDialog::PreferenceDialog(QWidget* parent)
       for (int idx = 0; idx < n; ++idx)
             exportAudioSampleRate->addItem(QString("%1").arg(exportAudioSampleRates[idx]));
 
-      restartWarningLanguage->setText("");
+      restartWarningLanguage->hide();
```

Comment:
yes I did test, it does change the layout (moves the button below a bit furter down), but doesn't change the overall size of the dialog, and that moving buton is sort of an eye-catcher so it is less easy to overlook that message